src/pages/home/HomePage.tsx

```diff
@@ -1,12 +1,168 @@
+								<div className="text-left text-sm">
+                                    12:00 <br className=""/>
+                                    18:00
+								</div>
+							</div>
+
+							<div className="flex justify-left items-center">
+								<CalendarIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    5 Mon
+								</div>
+							</div>
+							<div className="flex justify-left items-center">
+								<LocationMarkerIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    A1 - F2
+								</div>
+							</div>
+						</div>
+					</div>
+					<div className="rounded-md bg-white w-36 px-6 py-12 drop-shadow-md">
+						<div className="text text-center font-SofiaProBold grid grid-rows-3">
+							<div className="absolute right-0 top-0 p-3 hidden">
+								<StatusOnlineIcon className="h-7 w-7 text-red"/>
+							</div>
+							<div className="flex justify-left items-center">
+								<ClockIcon className="h-7 w-7"/>
+								<div className="text-left text-sm">
+                                    12:00 <br className=""/>
+                                    18:00
+								</div>
+							</div>
+
+							<div className="flex justify-left items-center">
+								<CalendarIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    5 Mon
+								</div>
+							</div>
+							<div className="flex justify-left items-center">
+								<LocationMarkerIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    A1 - F2
+								</div>
+							</div>
+						</div>
+					</div>
+					<div className="rounded-md bg-white w-36 px-6 py-12 drop-shadow-md">
+						<div className="text text-center font-SofiaProBold grid grid-rows-3">
+							<div className="absolute right-0 top-0 p-3 hidden">
+								<StatusOnlineIcon className="h-7 w-7 text-red"/>
+							</div>
+							<div className="flex justify-left items-center">
+								<ClockIcon className="h-7 w-7"/>
+								<div className="text-left text-sm">
+                                    12:00 <br className=""/>
+                                    18:00
+								</div>
+							</div>
+
+							<div className="flex justify-left items-center">
+								<CalendarIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    5 Mon
+								</div>
+							</div>
+							<div className="flex justify-left items-center">
+								<LocationMarkerIcon className="h-7 w-7"/>
+								<div className="text-lg ">
+                                    A1 - F2
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+
+			<div className="p-4 block fixed bottom-16 inset-x-0">
+				<button className="rounded-full bg-purple text-white p-4 w-full">Save Changes</button>
+			</div>
+
+			<div className="block fixed bottom-0 inset-x-0 p-4 grid grid-cols-3 gap-4 border-t-2 border-light-gray">
```

Comment:
Verwerkt dit alsjeblieft in een apart "Navbar" component, zodat deze herbruikt kan worden 😄